lab-3.py

```diff
@@ -0,0 +1,53 @@
+"""
+Runs a command and logs it to S3
+Usage:
+  lab-3.py something
+"""
+
+import subprocess
+import sys
+import boto3
+from datetime import datetime, timezone
+
+PROFILE="workshop"
+BUCKET="aws-workshop-bad-logs"
+
+boto3.setup_default_session(profile_name=PROFILE)
+s3 = boto3.client("s3")
+
+
+def main(argv):
+    cmd = None
+    if argv is None or len(argv) == 0:

```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To check if a container or sequence (string, list, tuple) is empty, use `if not val`. Do not compare its length using `if len(val) == 0` or `if len(val) &gt; 0`

 [Learn more](https://www.python.org/dev/peps/pep-0008/#programming-recommendations#:~:text=if%20not%20seq)